src/server/routes.rs

```diff
@@ -22,7 +29,12 @@ where
     Authn: crate::authn::Authenticator + Clone + Send + Sync + 'static,
     Authz: crate::authz::Authorizer + Clone + Send + Sync + 'static,
 {
-    let health = warp::path("healthz").map(|| "OK");
+    let health = warp::path("healthz").map(|| {
+        warp::reply::json(&HealthStatus {
+            status: "OK".to_string(),
+            version: clap::crate_version!().to_string(),
```

Comment:
This can be swapped out to a non-clap thing by doing `env!("CARGO_PKG_VERSION")`

Reply:
Updated it. Thank you.